extension/content.js

```diff
@@ -227,6 +227,33 @@ document.addEventListener('DOMContentLoaded', () => {
 		window.addEventListener('resize', infinitelyMore);
 	}
 
+	const details = $(`.vcard-details .vcard-detail`);
+	const isProfile = details.length > 0;
+	if (isProfile) {
+		const rhandle = new RegExp(`\\bhttps?:\\/\\/twitter\\.com\\/([A-z0-9_]+)\\b`, 'i');
+		const existing = details
+			.get()
+			.map(el => ({el, matches: rhandle.exec(el.innerText)}))
+			.filter(x => x.matches);
+
+		existing.forEach(x => {
+			const $el = $(x.el);
+			addTwitterProfile(x.matches[1], $el);
+			$el.remove();
+		});
+
+		if (!existing.length) {
+			fetch(`https://github-npm-profile.herokuapp.com/${ownerName}`)
```

Comment:
@bevacqua You may try the Twitter API and see if there's a Twitter user with the same handle as the user on GitHub.